File: app/src/main/java/com/yujin/fontchecker/model/FontModel.java
package com.yujin.fontchecker.model;

import androidx.lifecycle.MutableLiveData;
import com.yujin.fontchecker.R;

public class FontModel {
    public MutableLiveData<String>  text = new MutableLiveData<>();
    public MutableLiveData<Integer> font = new MutableLiveData<>();

    public FontModel() {
        text.setValue("");
        font.setValue(R.font.notosans_cjk_kr_regular);
    }
}

File: app/src/main/java/com/yujin/fontchecker/model/FontModel.kt
package com.yujin.fontchecker.model

import androidx.lifecycle.MutableLiveData
import com.yujin.fontchecker.util.DEFAULT_FONT
import com.yujin.fontchecker.util.DEFAULT_TEXT

class FontModel(_text: String? = DEFAULT_TEXT, _font: Int? = DEFAULT_FONT) {
    val text = MutableLiveData<String>(_text)
    val font = MutableLiveData<Int>(_font)
}